pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -79,6 +79,7 @@ const (
                    reverseThroughputFromDestinationNode) 
                    VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, 
                            ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+	//ttlQuery = `ALTER TABLE flows MODIFY TTL timeInserted + INTERVAL ? SECOND`
```

Comment:
Debug comment? Or move to `UpdateTTL` function if you think it's useful.

Reply:
Yes, it's a comment for debug. I've deleted. Thanks.